src/Tizen.NUI/src/public/WebView/WebView.cs

```diff
@@ -476,7 +517,7 @@ public int DefaultFontSize
         /// Event for the PageLoadStarted signal which can be used to subscribe or unsubscribe the event handler.<br />
         /// This signal is emitted when page loading has started.<br />
         /// </summary>
-        [EditorBrowsable(EditorBrowsableState.Never)]
+        /// <since_tizen> 9 </since_tizen>
```

Comment:
Please keep [EditorBrowsable(EditorBrowsableState.Never)] in DevelNUI branch.
When this patch is merged, you can remove it on Master branch with ACR.